kernel/src/misc/wm/wm.c

```diff
@@ -215,18 +215,19 @@ void wm_raise_window(wm_window_t* win) {
  * TODO: revamp entirely.
  */
 void wm_assign_position(wm_window_t* win) {
+    win->pos = (point_t*) kmalloc(sizeof(point_t));
```

Comment:
sorry I didn't catch that earlier, the window position shouldn't be a pointer.

Rationale is simply that memory allocation is way more complicated (see malloc.c) than stack, and pointers themselves are known to cause complications (see "returning kernel pointer from syscall" from previous review), and all this is in the kernel (admittedly dumb and my fault for the WM), which is not a place you want to complicate without good reason.